src/main/java/frc/robot/Robot.java

```diff
@@ -57,16 +57,21 @@
   private static final double highSpeed = 0.75;
   private static final double defaultSpeed = 0.65;
 
-  // Color Sensing
+  // Control Panel
   private final I2C.Port i2cPort = I2C.Port.kOnboard;
   private final ColorSensorV3 colorSensor = new ColorSensorV3(i2cPort);
   private final ColorMatch colorMatcher = new ColorMatch();
   private static final Color kBlueTarget = ColorMatch.makeColor(0.143, 0.427, 0.429);
   private static final Color kGreenTarget = ColorMatch.makeColor(0.197, 0.561, 0.240);
   private static final Color kRedTarget = ColorMatch.makeColor(0.561, 0.232, 0.114);
   private static final Color kYellowTarget = ColorMatch.makeColor(0.361, 0.524, 0.113);
+  private final WPI_TalonSRX controlPanelTalon = new WPI_TalonSRX(6);
   private String targetControlPanelColor = "N/A";
+  private String detectedColorString;
+  private String lastDetectedColorString;
   private int controlPanelSpinAmount = 0;
+  private int doubleContolPanelSpinAmount = controlPanelSpinAmount * 2;
```

Comment:
This should probably also be initialized to just 0. The doubling should be done when `controlPanelSpinAmount` is set. Really, I might go as far as to just double `controlPanelSpinAmount` *itself* when it's set.

Reply:
This variable has been removed in a recent commit.